frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditComponentIdRow/EditComponentIdRow.tsx

```diff
@@ -58,7 +62,10 @@ export const EditComponentIdRow = ({
   };
 
   return (
-    <div className={classes.container}>
+    <div
+      data-testid={'edit-component-id-row-container'}
```

Comment:
I think we can delete this data-testid, it's not used anymore?

Reply:
deleted